server/schedule/operator/step.go

```diff
@@ -38,6 +38,14 @@ const (
 	// DefaultFastExecutorRate is the slow rate of the operator executor.
 	// default:  0.1 s/Mb
 	DefaultFastExecutorRate = 0.1
+	// FastOperatorWaitTime is the duration that when an operator that is not marked
```

Comment:
Should be changed to XXXStepWaitTime? Also, the comment needs to be changed.

Reply:
fixed